packages/core/src/js-handlebars.ts

```diff
@@ -11,4 +11,7 @@ handlebars.registerHelper('json-stringify', function (input: any, indent?: numbe
 handlebars.registerHelper('eq', function (a: any, b: any) {
   return a === b;
 });
-export const compile = handlebars.compile;
+
+export function compile(input: string) {
```

Comment:
Technically, this changes the signature of this function, as you _could_ previously pass custom options. But this was nowhere the case AFAICT. And it _seems_ better to me to not expose this, but own the config internally here.

I can change that of course in case of disagreement!